examples/zephyr/certificate_provisioning/README.md

```diff
@@ -0,0 +1,182 @@
+# Golioth Certificate Provisioning Sample
+
+## Overview
+
+This sample application demonstrates one method for provisioning
+certificates onto a device for use in DTLS authentication. Certificates
+are loaded into the device's filesystem using ``mcumgr``.
+
+## Requirements
+
+* Golioth credentials
+* Network connectivity
+* A filesystem
+* ``mcumgr`` CLI tool
+
+## Building and Running
+
+### Platform specific configuration
+
+#### nRF52840 DK + ESP32-WROOM-32
+
+This subsection documents using nRF52840 DK running Zephyr with
+offloaded ESP-AT WiFi driver and ESP32-WROOM-32 module based board (such
+as ESP32 DevkitC rev. 4) running WiFi stack. See [AT Binary
+Lists](https://docs.espressif.com/projects/esp-at/en/latest/AT_Binary_Lists/index.html)
+for links to ESP-AT binaries and details on how to flash ESP-AT image on
+ESP chip. Flash ESP chip with following command:
+
+```console
+esptool.py write_flash --verify 0x0 PATH_TO_ESP_AT/factory/factory_WROOM-32.bin
+```
+
+Connect nRF52840 DK and ESP32-DevKitC V4 (or other ESP32-WROOM-32 based
+board) using wires:
+
+| nRF52840 DK | ESP32-WROOM-32  | ESP32-WROVER-32 |
```

Comment:
Let's also add the pin mapping for ESP32-C3-MINI-1 as in https://github.com/golioth/golioth-firmware-sdk/pull/165